01-JS-DOM-APIs/Practice1-1/Script/script.js

```diff
@@ -0,0 +1,43 @@
+function load () {
+  console.log('load'); //Only to know the page is loaded
+  let change = document.getElementById("changeclass");
+  change.style.transition = "all 3s ease-in-out";
+  change.style.opacity = 1;
+}
+
+
+function configAjax (methodHttp, url, asyncronic){
+  this.methodHttp = methodHttp;
+  this.url = url;
+  this.asyncronic = asyncronic;
+}
+
+function request(configAjax) {
```

Comment:
you need to pass *resolve* and *reject* functions as parameters as well, otherwise this function won't work